shim.c

```diff
@@ -1391,6 +1391,65 @@ uninstall_shim_protocols(void)
 #endif
 }
 
+EFI_STATUS
+load_sbat_level_file(EFI_HANDLE image_handle, CHAR16 *PathName)
+{
+	EFI_STATUS efi_status = EFI_SUCCESS;
+	PE_COFF_LOADER_IMAGE_CONTEXT context;
+	EFI_IMAGE_SECTION_HEADER *Section;
+	void *pointer;
+	int datasize = 0;
+	void *data = NULL;
+	int i;
+	char *sbat_var_previous = NULL;
+	char *sbat_var_latest = NULL;
+
```

Comment:
The read_image(), verify_image() could be factored out and re-used by load_cert_file(), let me know if you prefer that, it's ultimately a short function with a ton of arguments.